.github/workflows/workspace.yml

```
@@ -21,7 +21,7 @@ jobs:
     timeout-minutes: 45
     steps:
       - uses: actions/checkout@v4
-      - uses: dtolnay/rust-toolchain@1.74.0
+      - uses: dtolnay/rust-toolchain@1.78.0
```

Hmm, I think `integer_division_remainder_used` is shipping in Rust 1.79.0 on June 13th.

In the meantime it should work if you use `beta` for now (and maybe leave a comment to switch to Rust 1.79 after June 13th)

```suggestion
      - uses: dtolnay/rust-toolchain@beta
```

---

Will cut an issue to track change back to 1.79 also.